app/components/pay-page/choose-membership-plan-modal.hbs

```diff
@@ -0,0 +1,173 @@
+<ModalBody @allowManualClose={{true}} @shouldHideCloseButton={{true}} @onClose={{@onClose}} class="w-full">
+  <div class="mb-6">
+    <div class="font-bold text-2xl text-gray-600 text-center text-balance">
+      Invest in the membership that pays for itself many times over.
+    </div>
+
+    <p class="text-gray-700 text-sm text-center text-balance mt-2">
+      CodeCrafters members routinely unlock promotions and 5-figure salary jumps.
+    </p>
+  </div>
+
+  <AnimatedContainer>
+    {{#animated-if (eq this.previewType "plan") use=this.transition duration=200}}
+      <div>
```

Comment:
Extremely long file, let's split this into components

Reply:
@Arpan-206 think this was missed, I'd recommend splitting up the steps here into different components